src/event.rs

```diff
@@ -391,7 +391,7 @@ pub enum WindowEvent<'a> {
     ///
     /// ## Platform-specific
     ///
-    /// - **iOS / Android / Web:** Unsupported.
+    /// - **iOS / Android / Web / Redox OS:** Unsupported.
```

Comment:
I think using `Orbital` would be better? Maybe you'll have working Wayland platform at some point :p

Reply:
Yep, good point